docs/lib/datastore/fragments/native_common/datastore-events.md

```diff
@@ -1,14 +1,102 @@
 Amplify uses `Hub` for different categories to communicate with one another when specific events occur. DataStore emits `Hub` messages for the following events:
 
+## networkStatus
+
+Dispatched when DataStore starts and every time network status changes
+
+HubPayload `NetworkStatusEvent` contains a boolean value `active` to notify network status
+```
+{ active: true }
+```
```

Comment:
I think putting JSON blocks doesn't make much sense from an iOS and Android perspective. I think the list structure you have below in the `modelSynced` could be used for all events.

Reply:
Updated